src/django/api/admin.py

```
@@ -173,6 +173,21 @@ def get_readonly_fields(self, request, obj=None):
             return []
 
 
+class FacilityDownloadLimitAdmin(admin.ModelAdmin):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)